app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/compose/fakes/ComposePreviewFakes.kt

```diff
@@ -1,23 +0,0 @@
-package com.bottlerocketstudios.brarchitecture.ui.compose.fakes
```

Comment:
Thanks for deleting this file (I was not happy with the approach used prior but needed a way to render fullscreen previews). @BottleRocket-Colin's approach that you've implemented is much nicer 😄